stable/parse/values.yaml

```diff
@@ -0,0 +1,115 @@
+## Kubernetes serviceType for Parse Deployment
+## ref: http://kubernetes.io/docs/user-guide/services/#publishing-services---service-types
+##
+serviceType: LoadBalancer
+
+## loadBalancerIP for the Parse Service (optional, cloud specific)
+## ref: http://kubernetes.io/docs/user-guide/services/#type-loadbalancer
+##
+loadBalancerIP:
+
+parse:
```

Comment:
Values are grouped under `parse` and `parseDashboard`. I think it would make sense to use `server` and `dashboard` which reflect the component names.